.github/workflows/ci.yaml

```diff
@@ -1,68 +1,67 @@
-name: PyTest
+name: CI
 
 on: [push, pull_request]
```

Comment:
i've noticed we're running things twice when we do this, both for push and for PR.

Reply:
<img width="907" alt="Screen Shot 2022-06-14 at 5 27 31 PM" src="https://user-images.githubusercontent.com/3428642/173692207-1d424f0a-f656-4507-a4ed-0d57de5931af.png">

Reply:
i think we can solve it with something like the following:

```
on:
    push:
        branches: main
    pull_request:
```

Reply:
I saw this was happening as well, thought it was strange because I think the `on: [push, pull_request]` was a carryover from the previous version. 

I incorporated your suggestion and it seems to be working correctly now.